public/js/controllers/charts_controller.js

```diff
@@ -141,15 +141,16 @@ function nightModeOptions (nightModeOn) {
   }
 }
 
-function zipXYData (gData, isHeightAxis, isDayBinned, coefficient, windowS) {
-  coefficient = coefficient || 1
+function zipXYData (gData, isHeightAxis, isDayBinned, coefficient, windowS, initValue) {
```

Comment:
No need for this if you prepend the zeros.

Reply:
On this I think `initValue` makes sense as plotting zero missed votes before the stake validation height doesn't sound so good.

Reply:
If you really want to truncate the zeros, do it in the `ChartMaker`. That's the place for pre-processing of datasets.